addons/rhs/CfgVehicles.hpp

```diff
@@ -505,4 +505,23 @@ class CfgVehicles
     // ### RHS inheritance fix
     class rhs_uaz_spg9_base;
     class rhs_uaz_spg9_chdkz : rhs_uaz_spg9_base {};
+
+    /*// UGV Stomper
+    class UGV_01_base_F;
+    class UGV_01_rcws_base_F : UGV_01_base_F
```

Comment:
Wir machen wieder eine TB Variante draus damit wir die Original Sachen nicht verändern ;D

Reply:
In Ordnung, dankesehr!